test/test_method.vim

```diff
@@ -0,0 +1,5 @@
+echo 'alice'->s:hello()
+echo 'alice'->s:hello('bob')
+echo [1,2,3]->map({i,n -> n * 2})
+echo 'john'->{s -> 'hello ' .. s }()
+echo 'john'->{s,b -> 'hello ' .. s .. ' ... and goodbye ' .. b .. '!' }('bob')
```

Comment:
please add one more test using `...` for argument.

Reply:
fixed e6c1c4c5ee162f1ca28c4041b5193b622428a3e8